authors/apps/articles/tests/tests_views.py

```diff
@@ -0,0 +1,73 @@
+from django.test import Client
+from django.test import TestCase
+from authors.apps.authentication.models import User
+import json
+
+
+class BaseTest(TestCase):
+
+    def setUp(self):
+        """ automatically run the function before running other tests"""
+        self.test_client = Client()
+
+        self.article_to_create = {
+            "article": {
+                "title": "hahahhaha hahaha hahah ahhah",
+                "body": "<p>this is a body that is here hahahhahaha\n <p",
```

Comment:
@abulojoshua1 
Can you clarify on why you are passing in raw HTML here

Reply:
i think this is some data to be used in the test and it should be okay to write some raw html

Reply:
I would like to mimic the presence on an article that has HTML attributes like *Bolding*, paragraphs, etc in an article body. But as for now, that for tests.